bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/details/LineItemDetailsListener.kt

```diff
@@ -9,9 +9,9 @@ import net.gini.android.bank.sdk.capture.digitalinvoice.SelectableLineItem
  */
 
 /**
- * Interface used by the [LineItemDetailsFragment] to dispatch events to the hosting Activity.
+ * Interface used by the [DigitalInvoiceActivity] to dispatch events to it's fragment.
  */
-interface LineItemDetailsFragmentListener {
+interface LineItemDetailsListener {
```

Comment:
[Request] Can be `internal` as well or add `Internal use only.`.

Reply:
[Note]: Made interfaces internal but then had to make the "internal use only" DigitalInvoiceBottomSheet class too, because it exposed the variable from the interface.  Please check if that is good or the listener interface should remain public with the Internal comment...

Reply:
That is great, thanks!